plugin/trino-geospatial/src/main/java/io/trino/plugin/geospatial/GeometryType.java

```diff
@@ -14,22 +14,38 @@
 package io.trino.plugin.geospatial;
 
 import io.airlift.slice.Slice;
+import io.airlift.slice.XxHash64;
 import io.trino.spi.block.Block;
 import io.trino.spi.block.BlockBuilder;
 import io.trino.spi.block.VariableWidthBlock;
 import io.trino.spi.block.VariableWidthBlockBuilder;
 import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.function.IsNull;
+import io.trino.spi.function.ScalarOperator;
 import io.trino.spi.type.AbstractVariableWidthType;
 import io.trino.spi.type.StandardTypes;
+import io.trino.spi.type.TypeOperatorDeclaration;
+import io.trino.spi.type.TypeOperators;
 import io.trino.spi.type.TypeSignature;
 
 import static io.trino.geospatial.serde.GeometrySerde.deserialize;
+import static io.trino.spi.function.OperatorType.EQUAL;
+import static io.trino.spi.function.OperatorType.HASH_CODE;
+import static io.trino.spi.function.OperatorType.IDENTICAL;
+import static io.trino.spi.function.OperatorType.XX_HASH_64;
 
 public class GeometryType
         extends AbstractVariableWidthType
 {
     public static final GeometryType GEOMETRY = new GeometryType();
 
+    private static final TypeOperatorDeclaration TYPE_OPERATOR_DECLARATION =
+            TypeOperatorDeclaration.builder(Slice.class)
+                    .addOperators(DEFAULT_READ_OPERATORS)
+                    .addOperators(DEFAULT_COMPARABLE_OPERATORS)
+                    .addOperators(DEFAULT_ORDERING_OPERATORS)
```

Comment:
Ordering doesn't make sense for geometries.

Reply:
gotcha, @martint . addressed.